### PR TITLE
A new gulp base for a new year

### PR DESCRIPTION
**Pull request by davidblurton**

### CODE REVIEW

File: gulp/tasks/watch.js [hunk 1]
```diff
@@ -1,13 +1,30 @@
...
+      gulp.start('copy');
+    });
+
+    watch(gulp.config.source + '/views/**/*.jade', function() {
+      gulp.start('jade');
+    });
+
+    watch(gulp.config.source + '/img/**/*.{png,gif,jpg,jpeg,svg}', function() {
+      gulp.start('images');
+    });
```
Comment (eirikurn) [inline gulp/tasks/watch.js:28]: Why use `gulp-watch` instead of `gulp.watch`? Seems like we're not even using it's fancy streaming features.

**Comment by eirikurn** [comment]
Cool! Just one comment.

File: gulp/tasks/watch.js (hunk 1 above)
Comment (davidblurton) [inline gulp/tasks/watch.js:28]: It watches new files without restarting.

Comment (eirikurn) [inline gulp/tasks/watch.js:28]: Ah, fair enough

Comment (davidblurton) [inline gulp/tasks/watch.js:28]: Might be able to go back to the native gulp.watch when gulp 4 ships.

**Pull request merged**